polkadot-protocol/architecture/polkadot-chain/agile-coretime.md

```diff
@@ -1,4 +1,48 @@
 ---
 title: Agile Coretime
-description: TODO
----
\ No newline at end of file
+description: Explore the efficient scheduling mechanisms to access Polkadot cores to produce blockspace continuously or on-demand.
+---
+
+# Agile Coretime
+
+Agile Coretime is the [scheduling](https://en.wikipedia.org/wiki/Scheduling_(computing)){target=\_blank} 
+framework on Polkadot that lets parachains efficiently access cores, which comprise an 
+active validator set tasked with parablock validation. Polkadot is the first blockchain to
+enable such a flexible scheduling framework for producing blockspace. The cores can be designated
+to a parachain continuously through [bulk coretime](#bulk-coretime) or [on-demand](#on-demand-coretime).
+It is also possible to schedule multiple cores in parallel through elastic scaling, which is
```

Comment:
Maybe we can add a link to somewhere else that explains a bit further about elastic scaling. WDYT @DrW3RK?

Maybe https://github.com/paritytech/polkadot-sdk/issues/1829? Or maybe https://wiki.polkadot.network/docs/learn-elastic-scaling?

I'd prefer the wiki since a GitHub issue does not seem like the best place to look for this info